tests/test_helpers.py

```diff
@@ -860,6 +833,26 @@
 #      "List of all tags in citations collection:\n['nomonth', 'pdf']\nbuilding lists for all tags in the citation collection\nnomonth has been added/updated in reading_lists\npdf has been added/updated in reading_lists\n"),
 #     ]
 
+helper_map_bad = [
+ (["helper", "u_milestone", "--milestone_uuid", "sb_fir", "--projectum_id", "sb_firstprojectum",
+   "--due-date", "2023-01-01", "--notes", "do this", "do that", "--type", "mergedpr"],
+  "A uuid fragment and projectum id have been entered.\n"
+  "You may enter a milestone uuid or a projectum id but not both.\n"
+  "Enter a milestone uuid to update an existing milestone, or a projectum id to add a new milestone to that projectum.\n"
+  ),
+ (["helper", "u_milestone", "--due-date", "2023-01-01",
+   "--notes", "do this", "do that", "--type", "mergedpr"],
```

Comment:
remove these.  Just put enough to test the thing you are testing.  probably, `["helper", "u_milestone"]`

This makes it easier for someone to understand your intent with fewer brain cells burning

Reply:
:+1:

Reply:
makes sense